Add css and jsx to products page and remake data fetching using react-query instead of loaders  with useRouteLoaderData useLoaderData

Moriartymath opened the pull request.

Moriartymath committed:
Add basic css and jsx to products page
- Add basic jsx and css to cardItem
- Move loaders.tsx to utils

Moriartymath committed:
Add basic css and jsx to products page
- Add basic jsx and css to cardItem
- Move loaders.tsx to utils

Moriartymath committed:
Add basic css and jsx to products page
- Add basic jsx and css to cardItem
- Move loaders.tsx to utils

Moriartymath committed:
Add sort context, and more css

Moriartymath committed:
Add sort context, and more css

Moriartymath committed:
Add sort context, and more css

Moriartymath committed:
Create files and add more css

Moriartymath committed:
Create files and add more css

Moriartymath committed:
Create files and add more css

Moriartymath committed:
Add 3 filters

Moriartymath committed:
Add 3 filters

Moriartymath committed:
Add 3 filters

Moriartymath committed:
Add  size filter with css
- Add media query to hide filters whenever width <= 1250 px

Moriartymath committed:
Add  size filter with css
- Add media query to hide filters whenever width <= 1250 px

Moriartymath committed:
Add  size filter with css
- Add media query to hide filters whenever width <= 1250 px

TarasBlatnoi reviewed (changes requested):
![image](https://github.com/user-attachments/assets/e0532d00-40fc-44a1-b1a3-c4902a8c7965)
Новинки should drop out when i click on сортування it shouldn't be visible immidiately, checkboxes is wrong, columns of size in filter should be 3 not 5 
![image](https://github.com/user-attachments/assets/a7a3f97e-9145-4a43-a698-3c3562aec9a6)
when i press hide filter products should be on all screen they need to stretch not just that filter hides
![image](https://github.com/user-attachments/assets/8c86b080-a79a-45d4-afc8-ab7c67d35ef9)
here should be space between border bottom and horizontal line on the right

Moriartymath committed:
Add more css
 - Add height to main
 - Make filters to be sticky
 - Add font

Moriartymath committed:
Add more css
 - Add height to main
 - Make filters to be sticky
 - Add font

Moriartymath committed:
Add custom checkBox component, and more css
- Add formating for prices

Moriartymath committed:
Add custom checkBox component, and more css
- Add formating for prices

Moriartymath committed:
Add more css
- Add sort drop down

Moriartymath committed:
Add more css
- Add sort drop down

Moriartymath committed:
Add Filters provider and more css

Moriartymath committed:
Add Filters provider and more css

Moriartymath committed:
Add font styling to gender name

Moriartymath committed:
Add font styling to gender name

Moriartymath committed:
small css change

Moriartymath committed:
small css change

Moriartymath committed:
Remake animation of products

Moriartymath committed:
Remake animation of products

Moriartymath committed:
add media queries

Moriartymath committed:
add media queries

Moriartymath committed:
Change products css
- Fix height of card item
- Hiding filters at the end of animation

Moriartymath committed:
Change products css
- Fix height of card item
- Hiding filters at the end of animation

Moriartymath committed:
Change css
- Add sticky position to top container
- Change visibility of hidden filters

Moriartymath committed:
Change css
- Add sticky position to top container
- Change visibility of hidden filters

Moriartymath committed:
Remake fetching data with react query

Moriartymath committed:
Remake fetching data with react query

TarasBlatnoi committed:
Merge branch 'illia' of https://github.com/TarasBlatnoi/TerabadeReact into illia

TarasBlatnoi committed:
Merge branch 'illia' of https://github.com/TarasBlatnoi/TerabadeReact into illia

Moriartymath committed:
Refactor routes
- Change sql queries to instead of male and female ( sex propery) now its men and women

Moriartymath committed:
Change css
- Fix transition duration
- Add to "сортування" animated css figure instead of image
- Change font, and spacing
-  Change label for "приховати фільтр" to "показати фільтр" when toggle show/hide filter
- Add cursor pointer to card items, filters and toggle buttons

Moriartymath committed:
Add you might also like section

Moriartymath committed:
Change css of cardItem

Moriartymath committed:
Fix some css

Moriartymath committed:
Implement section scroll functionality

Moriartymath committed:
Merge branch 'main' into illia
